packages/components/src/QualityBar/QualityBar.stories.js

```diff
@@ -12,21 +12,51 @@ stories
 				<header>Quality Bar</header>
 				<div>
 					<div>Homogeneous Quality</div>
-					<QualityBar invalid={30} valid={30} empty={30}></QualityBar>
+					<QualityBar invalid={30} valid={30} empty={30} />
 					<div>Very invalid</div>
-					<QualityBar invalid={30} valid={0} empty={0}></QualityBar>
+					<QualityBar invalid={30} valid={0} empty={0} />
 					<div>Best quality ever</div>
-					<QualityBar invalid={0} valid={30} empty={0}></QualityBar>
+					<QualityBar invalid={0} valid={30} empty={0} />
 					<div>Nothing to see here</div>
-					<QualityBar invalid={0} valid={0} empty={30}></QualityBar>
+					<QualityBar invalid={0} valid={0} empty={30} />
 					<div>Invalid and Empty</div>
-					<QualityBar invalid={0} valid={30} empty={30}></QualityBar>
+					<QualityBar invalid={0} valid={30} empty={30} />
 					<div>Classic look</div>
-					<QualityBar invalid={2} valid={88} empty={3}></QualityBar>
+					<QualityBar invalid={2} valid={88} empty={3} />
 					<div>Classic look (again yep)</div>
-					<QualityBar invalid={122} valid={1088} empty={293}></QualityBar>
+					<QualityBar invalid={122} valid={1088} empty={293} />
 					<div>I really like the digits !</div>
-					<QualityBar invalid={30} valid={30} empty={30} digits={5}></QualityBar>
+					<QualityBar invalid={30} valid={30} empty={30} digits={5} />
+					<div>Classic look with action button</div>
+					<QualityBar
+						invalid={2}
+						valid={88}
+						empty={3}
+						getDataAction={qualityType => {
```

Comment:
I think it would be better to have something like this

<QualityBar
   invalid={2}
   valid={88}
   empty={3}
   onClick={(event, { type: 'valid' | 'empty'..., buttonRef}) => {})
   dataFeature="preparation.filter.column." // <- or a dataFeature={type => `preparation.filter.column.${type}`}
 />
 
 It's easier to read and use
 
What do you think?

Reply:
@romainseb  ☝️ WDYT?

Reply:
Looks good this way

Reply:
ok, it's in progress